c7n/resources/kms.py

```diff
@@ -23,13 +23,21 @@
 
 class KeyBase(object):
 
+    permissions = ('kms:ListResourceTags',)
+
     def augment(self, resources):
         client = local_session(
             self.session_factory).client('kms')
         for r in resources:
             key_id = r.get('AliasArn') or r.get('KeyArn')
             info = client.describe_key(KeyId=key_id)['KeyMetadata']
             r.update(info)
+
+            tags = client.list_resource_tags(KeyId=key_id)['Tags']
```

Comment:
same question here, wrt to why a resource specific tag api instead of the resource group tag api.